setup_venv3.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -e
+
+venv_path=".venv3"
+
+pip install --user virtualenv
```

Comment:
`python3 -m venv` could preclude this. Also, if this is an app, maybe `pipenv` would be a good solution?

Reply:
Yes, I want to change the packaging etc. and am strongly considering deprecating Python 2 support altogether (given the impending support deadline). This was something of a quick hack in line with the existing script.